account_invoice_line_report/report/account_invoice_report_view.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2017 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_invoice_report_tree_info" model="ir.ui.view">
+        <field name="name">account.invoice.report.tree.info</field>
+        <field name="model">account.invoice.report</field>
+        <field name="arch" type="xml">
+            <tree string="Invoice Line">
+                <field name="partner_id" />
+                <field name="product_categ_id" />
+                <field name="product_id" />
+                <field name="quantity" />
+                <field name="price_average" />
+                <field name="price_subtotal" />
+                <field name="currency_id" invisible="1" />
+            </tree>
+        </field>
+    </record>
+    <record id="view_account_invoice_report_search" model="ir.ui.view">
+        <field name="model">account.invoice.report</field>
+        <field name="inherit_id" ref="account.view_account_invoice_report_search" />
+        <field name="arch" type="xml">
+            <field name="invoice_date" position="before">
+                <field
+                    name="product_uom_id"
```

Comment:
I have some doubts about using this variable here. In v12, it used `uom_name`, which was removed in v13. I think this is doing the correct mapping, but I also don't understand very well the search filter it is doing below (why compare `partner_id` with `uom_name`?) so I'm not sure if it is doing all the functionality from the previous version...

Reply:
This is a trick to filter by "on fly related field"... Now we can use https://github.com/OCA/server-ux/tree/13.0/base_search_custom_field_filter so you can remove this filter field and the next.